tools/autograd/gen_python_functions.py

```diff
@@ -80,7 +80,6 @@
 
 # These functions require manual Python bindings or are not exposed to Python
 _SKIP_PYTHON_BINDINGS = [
-    "alias",
```

Comment:
Did you add proper documentation for this new python API? I don't see anything here?